translations/pl.json

```diff
@@ -578,6 +581,8 @@
     "sync-in-progress": "Synchronizacja...",
     "sync-synced": "W synchronizacji",
     "test-networks": "Sieci testowe",
+    "testfairy-message": "Używasz aplikacji zainstalowanej z nocnej wersji. Jeśli masz połączenie z Wi-Fi, Twoje interakcje z aplikacją zostaną zapisane jako wideo i logi. Te nagrania nie zapisują twoich haseł. Są one używane przez nasz zespół  do zbadania możliwych problemów i występują tylko wtedy, gdy aplikacja jest instalowana z nocnej wersji. Nic nie jest nagrywane, jeśli aplikacja jest zainstalowana z PlayStore lub TestFlight.",
```

Comment:
we should remove testfairy related keys

Reply:
Removed from Lokalise, as well as changed a config that should removed deleted keys.
This PR was auto-generated so I'm not going to manually edit it, but you are free to remove this if you like (but it seems safe to leave it in this PR)

Reply:
sounds good, we can do it later

Reply:
Cool thanks. 
Since it is deleted it won't be in the next release anyway.
About 10 other keys were removed too.